travis/scripts/04-tests.sh

```diff
@@ -46,5 +46,5 @@ elif [ -f "gradlew" ]; then
         -Dlogging.level.io.github.jhipster.travis=OFF
 fi
 if [ -f "tsconfig.json" ]; then
-    yarn test -u
+    yarn test
```

Comment:
With npm `npm test -- -u` should work and with yarn it will produce an extra warning which should be ok

Reply:
` npm test -- -u` works ! thanks guys

I will revert my commits and fix cicd sub gen accordingly.